README.md

```diff
@@ -55,6 +55,13 @@ Note: Nextjs uses `.env.development` and `.env.production` for their respective
 2. `npm install`
 3. `npm run dev` and point browser to http://localhost:3000/library-card/new
 
+#### NOTE
+
+When you type `npm run dev` the CLI will output a line:
```

Comment:
Nit: This output only happens when you are visiting localhost/3000 - not generally.